components/Trip/index.js

```diff
@@ -17,18 +18,40 @@ export default function Trip({
   city,
   country,
 }) {
+  function DisplayCountdown() {
+    const countdown = formatDistanceToNowStrict(new Date(startDate), {
+      unit: "day",
+      roundingMethod: "floor",
+      addSuffix: true,
+    });
+
+    const parsedCountdown = countdown.replace(/[^0-9]/g, "");
+
+    const countdownAdjusted = formatDistanceToNowStrict(new Date(startDate), {
+      roundingMethod: "floor",
+      addSuffix: true,
+    });
+
+    if (!countdown.includes("ago") && parsedCountdown < 30) {
+      return `starts ${countdownAdjusted}`;
+    }
+    if (countdown.includes("ago") && parsedCountdown < 1) {
+      return "starts today";
+    }
+  }
   return (
     <StyledLink href={`/trips/${id}`}>
       <StyledTrip>
         <StyledImage src={image} width={70} height={70} alt="" />
```

Comment:
This is not part of the PR, but I suggest to use a more meaningful alt description for the image to aid with accessibility.

```suggestion
        <StyledImage src={image} width={70} height={70} alt={`Image for trip titled ${title}`} />
```